kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/xstream/XStreamMarshaller.java

```diff
@@ -16,11 +16,11 @@
 package org.kie.server.api.marshalling.xstream;
 
 import java.util.HashMap;
-import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
 
 import com.thoughtworks.xstream.XStream;
+import com.thoughtworks.xstream.converters.reflection.PureJavaReflectionProvider;
```

Comment:
alright, that's fine just was wondering if we could initialize that differently than final/constructor way. But if you feel that's the best way then I am good with this. Assuming that this PureJavaReflectionProvider is public api so to say.